chart/elastalert2/templates/config.yaml

```diff
@@ -11,7 +11,7 @@ metadata:
 data:
   elastalert_config: |-
     ---
-    rules_folder: /opt/elastalert/rules
+    rules_folder: {{ default .Values.rulesFolder | default "/opt/elastalert/rules" }}
```

Comment:
What's the purpose of specifying this default here, and in the mountPath of deployment.yaml, when you've already defined the default in values.yaml?

Reply:
You are right, deployment should not be changed.

Reply:
That wasn't my point. I was asking why you are specifying the literal string `/opt/elastalert/rules` in three locations:
- config.yaml
- deployment.yaml
- values.yaml
when it only needs to be in the values.yaml file. Duplicating default values should be avoided.

In other words, I'm suggesting that this line should be:
```
    rules_folder: {{ .Values.rulesFolder }}
```

And line 60 of deployment.yaml should be:

```
            mountPath: {{ .Values.rulesFolder }}
```

Reply:
Sure, it can be done like that.
The reason I have done it like that was to make this change backwards compatible (no changes in values.yaml are required right now).

Updating the pr :)

Reply:
This shouldn't be a breaking change since the chart's included values.yaml file is not supposed to be modified. Instead, the user provides an overrides file during deployment, with their customizations.